enterprise/frontend/src/metabase-enterprise/content_translation/utils.ts

```diff
@@ -133,9 +134,34 @@ export const translateFieldValuesInSeries = (
     }
     const untranslatedRows = singleSeries.data.rows.concat();
 
-    const translatedRows = singleSeries.data.rows.map((row) =>
-      row.map((value) => tc(value)),
-    );
+    let translatedRows: RowValue[][];
+
+    if (singleSeries.card?.display === "pie") {
+      const pieRows = singleSeries.card.visualization_settings?.["pie.rows"];
```

Comment:
nit: I prefer `Object.fromEntries` for these things, makes the intent a little clearer, and maybe a bit more concise.

```suggestion
      const pieRows = singleSeries.card.visualization_settings?.["pie.rows"] ?? [];
      const keyToNameMap = Object.fromEntries(
        pieRows.map(row => [row.key, row.name])
      );
```